ctr-std/src/macros.rs

```diff
@@ -220,6 +220,23 @@ macro_rules! eprintln {
     })
 }
 
+#[stable(feature = "debug", since = "1.69.0")]
+#[macro_export]
+macro_rules! dbg {
+    ($val:expr) => {
+        // Use of `match` here is intentional because it affects the lifetimes
+        // of temporaries - https://stackoverflow.com/a/48732525/1063961
+        match $val {
+            tmp => {
+                eprintln!("[{}:{}] {} = {:#?}",
+                    file!(), line!(), stringify!($val), &tmp);
+                tmp
+            }
+        }
+    }
+}
+
+
```

Comment:
Missing doc comment prevents compilation from succeeding

```suggestion
/// A macro to await on an async call.
```